lib/screen/exhibitions/exhibitions_page.dart

```diff
@@ -76,6 +83,7 @@ class ExhibitionsPageState extends State<ExhibitionsPage> with RouteAware {
 
   void refreshExhibitions() {
     _exhibitionBloc.add(GetAllExhibitionsEvent());
+    _subscriptionBloc.add(GetSubscriptionEvent());
```

Comment:
calling GetSubscriptionEvent here doesn't make a lot of sense.
it should be called when starting the app and after update subscription status